[radiothermostat] Add units to README

mlobstein opened the pull request:
Add units and other minor readme edits.

jlaur commented on bundles/org.openhab.binding.radiothermostat/README.md:
```diff
@@ -169,10 +171,10 @@ Number Therm_Hour      "Thermostat Hour [%d]"                                { c
-Number:Time Therm_todaycool "Today's Cooling Runtime [%d %unit%]"       { channel="radiothermostat:rtherm:mytherm1:today_cool_runtime" }
-Number:Time Therm_yesterdayheat "Yesterday's Heating Runtime [%d %unit%]"   { channel="radiothermostat:rtherm:mytherm1:yesterday_heat_runtime" }
-Number:Time Therm_yesterdaycool "Yesterday's Cooling Runtime [%d %unit%]"   { channel="radiothermostat:rtherm:mytherm1:yesterday_cool_runtime" }
+Number:Time Therm_todayheat "Today's Heating Runtime [%d %unit%]"       { channel="radiothermostat:rtherm:mytherm1:today_heat_runtime", unit="min" }
+Number:Time Therm_todaycool "Today's Cooling Runtime [%d %unit%]"       { channel="radiothermostat:rtherm:mytherm1:today_cool_runtime", unit="min" }
```
Consider reformatting also now that you are at it:
```suggestion
Number:Time Therm_todayheat    "Today's Heating Runtime [%d %unit%]"       { channel="radiothermostat:rtherm:mytherm1:today_heat_runtime", unit="min" }
Number:Time Therm_todaycool    "Today's Cooling Runtime [%d %unit%]"       { channel="radiothermostat:rtherm:mytherm1:today_cool_runtime", unit="min" }
```

jlaur reviewed (commented):
Thanks! I have added one minor (optional) formatting comment not directly related to your changes.

Perhaps it would be worth mentioning the unit also in the table of channels? This will help in case of linking to a raw `Number` item.

Please note that there are also two temperature channels which could benefit from having the unit mentioned (°F).

mlobstein commented:
> Thanks! I have added one minor (optional) formatting comment not directly related to your changes.
> 
> Perhaps it would be worth mentioning the unit also in the table of channels? This will help in case of linking to a raw `Number` item.
> 
> Please note that there are also two temperature channels which could benefit from having the unit mentioned (°F).

I added (°F) to the three temperature channels in the table. If this is not what you were thinking, can you show an example?

jlaur commented:
> I added (°F) to the three temperature channels in the table. If this is not what you were thinking, can you show an example?

Perfect!

jlaur reviewed (approved):
LGTM
